surveys/views.py

```diff
@@ -342,14 +387,24 @@ def get_context_data(self, **kwargs):
 
 class SurveyListRun(ListView):
     model = SurveyFormEntry
-    template_name = "survey_list.html"
+    template_name = "survey_list_run.html"
     context_object_name = 'surveys'
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['surveys'] = context['surveys'].exclude(active=False).filter(published=True).order_by('-updated')
         context['published'] = True
 
+        for survey in context['surveys']:
+            survey_questions = fobi_models.FormElementEntry.objects.all().filter(form_entry_id=survey.id)
+            survey.question_count = len(survey_questions)
```

Comment:
Let's use [`count()`](https://docs.djangoproject.com/en/2.1/ref/models/querysets/#django.db.models.query.QuerySet.count) here instead of `len()`. From [the docs](https://docs.djangoproject.com/en/2.1/ref/models/querysets/):

> A QuerySet is evaluated when you call len() on it. This, as you might expect, returns the length of the result list.
>
> Note: If you only need to determine the number of records in the set (and don’t need the actual objects), it’s much more efficient to handle a count at the database level using SQL’s SELECT COUNT(*). Django provides a count() method for precisely this reason.